app/models/activity/import/field.rb

```diff
@@ -0,0 +1,80 @@
+class Activity::Import::Field
```

Comment:
How do we feel about the interface for this class? I feel I got a bit carried away

Reply:
I like this too - that `Activity::Import` class and its tests have been getting more and more unwieldy, so breaking them down into smaller chunks is much appreciated, and this feels like a good place to split out some of this functionality.